pyproject.toml

```diff
@@ -54,3 +54,6 @@ select = [
 ignore = [
     "E501",  # line too long
 ]
+exclude = [
+    "tests/lib/**"
```

Comment:
we don't want to run ruff on solady or vyper stuff we may import in external libs